server/container_create.go

```diff
@@ -176,7 +176,7 @@ func addImageVolumes(ctx context.Context, rootfs string, s *Server, containerInf
 				}
 			}
 		case config.ImageVolumesBind:
-			volumeDirName := stringid.GenerateNonCryptoID()
+			volumeDirName := stringid.GenerateRandomID()
```

Comment:
`GenerateNonCryptoID()` is an attempt to avoid exhausting the kernel's RNG, but it apparently wasn't yet in the version of `github.com/docker/docker` that we're pulling in.  Changing the `stringid` import to point to the copy in `github.com/containers/storage/pkg/stringid` should provide a `GenerateNonCryptoID()` implementation.

Reply:
Looking at `https://github.com/cri-o/cri-o/pull/2874/files#diff-ce66400d7362a285b3572807a6f3dd8b`, these changes adapt to the changed API without changing what actually happens.

(Replacing the dependency may well make sense, I’m just saying that it isn’t strictly necessary _right now_ to affect entropy.)

Reply:
If I'm reading that right, `GenerateRandomID()` continues to read from `crypto/rand`, but we lose the function that eventually reads from `math/rand`.

Reply:
Ah, you’re right, sorry. I have not noticed the `rand` import moving from `math/rand` to `crypto/rand`.

Reply:
Fixed, thanks!